modules/selection/src/main/kotlin/com.r3.corda.lib.tokens.selection/memory/config/InMemorySelectionConfig.kt

```diff
@@ -28,7 +28,7 @@ data class InMemorySelectionConfig @JvmOverloads constructor(val enabled: Boolea
             val cacheSize = config.getIntOrNull("stateSelection.inMemory.cacheSize")
                     ?: CACHE_SIZE_DEFAULT
             val indexingType = try {
-                (config.get("stateSelection.inMemory.indexingStrategies") as List<Any>).map { VaultWatcherService.IndexingType.valueOf(it.toString()) }
+                (config.get("stateSelection.inMemory.indexingStrategies") as List<*>).map { VaultWatcherService.IndexingType.valueOf(it.toString()) }
```

Comment:
Can you revert and add a @Suppress("UNCHECKED_CAST")

Reply:
Done